components/Eingabefeld 1/index.js

```diff
@@ -0,0 +1,59 @@
+import styled from "styled-components";
+import { locations } from "../../db/data.js";
+import moment from "moment";
+
+const now = moment();
+const lockedTime = now.set().format("HH:mm");
+
+export default function InformationPage() {
+  return (
+    <OverAllWrapper>
+      <DisplayFlex>
+        Kleidung <br /> hinterlegt
+        <DarkGreenContainer>
+          <p>
+            <strong>{lockedTime} Uhr</strong>
```

Comment:
Inline styling as in `<strong>{lockedTime} clock</strong>` should be avoided as it is difficult to maintain and change. CSS files or styled components should be used instead.